tool/junit/src/main/kotlin/flank/junit/mapper/Structural.kt

```diff
@@ -31,13 +31,17 @@ internal fun List<JUnit.TestResult>.mapToTestSuites(): List<JUnit.Suite> = this
 
 internal fun List<JUnit.Suite>.mapToTestResults(): List<JUnit.TestResult> =
     flatMap { suite ->
+        var startAt: Long
+        var endAt: Long = 0
         suite.testcases.map { case ->
+            startAt = endAt
+            endAt = startAt + (case.time * 1000).toLong()
             JUnit.TestResult(
                 suiteName = suite.name,
                 testName = case.name,
                 className = case.classname,
-                startAt = 0,
-                endsAt = (case.time * 1000).toLong(),
+                startAt = startAt.also { startAt = endAt },
```

Comment:
wont this simply result in 
endAt = endAt
& 
startAt = endAt?

Reply:
I don't understand your suggestion, but it's reminded me that I had forgotten to add a proper test. So, I added a small update. If you see a shorter solution, feel free to check out the branch and test it using `StructuralKtTest.mapToTestResultsTest`

Reply:
Right, this part `.also { startAt = endAt }` was not necessary